CHANGES.md

```diff
@@ -8,6 +8,12 @@
   access the `GAP` module reliably, use the GAP variable `GAP_jl` instead.
 - Add the GAP function `JuliaType`,
   for constructing parametrized Julia types in GAP
+- Remove the GAP filter `IsJuliaWrapper`.
```

Comment:
I think this has the potential of breaking user code, so shouldn't it be marked as breaking in the release notes?

Also, not just the filter is removed (this a user could easily deal with), but also the special treatment for it on the kernel level...

```suggestion
- **Breaking:** Remove the GAP filter `IsJuliaWrapper` and its special treatment.
```